api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/GoogleAnalyticsClient.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.dhsc.htbhf.claimant.reporting;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.dhsc.htbhf.claimant.exception.GoogleAnalyticsException;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.springframework.http.HttpMethod.POST;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.MediaType.TEXT_PLAIN;
+
+/**
+ * Service for interacting with google analytics measurement protocol.
+ * See https://developers.google.com/analytics/devguides/collection/protocol/v1/
+ */
+@Service
```

Comment:
Shouldn't this be a `@Component` rather than `@Service`?

Reply:
Changed to Component.